### ISSUE TITLE
UI: devices view fails when the example device has been deleted and no other non vpn only devices are present

### ISSUE THREAD

**Issue by lts-rad** [spr-networks/super#320]

**Comment by lts-rad** [comment]
page redirects to validate. no otp code hsa been set

<img width="652" alt="image" src="https://github.com/spr-networks/super/assets/37549748/92c6d961-1ea4-41c3-92c4-5b971cfd752b">

**Comment by lts-rad** [comment]
user reports this is not just virtual mode. 

looking into it, the only request that is failing is the ouis lookup which 404s. then the UI incorrectly handles the 404 to validate the otp code

**Comment by lts-rad** [comment]
bug does not appear to reproduce with this change

```
        if (macs && macs.length > 0) {
          // set device oui if avail
          deviceAPI
            .ouis(macs)
            .then((ouis) => {
              let devs = devices.map((d) => {
                let oui = ouis.find((o) => o.MAC == d.MAC)
                d.oui = oui ? oui.Vendor : ''
                return d
              })

              setList(devs.sort(sortDevices))
            })
            .catch((err) => {})
        }
```

but we need a more extensive fix

**Comment by lts-rad** [comment]
the issue is the empty lookup triggers the redirect http://127.0.0.1:8009/plugins/lookup/ouis/ which incorrectly assumes otp failure

**Comment by lts-rad** [comment]
this is fixed correctly by 18f7882c8e8972aa8ba35680c47d012f71a5b4ac
